telegram/constants.py

```diff
@@ -1843,6 +1843,21 @@ class MessageType(StringEnum):
 
     .. versionadded:: 20.8
     """
+    REPLY_TO_STORY = "reply_to_story"
```

Comment:
please keep the members in alphabetical order :) makes it so much easier to search for things :D

Reply:
no idea how that ended up there sry